### PR TITLE
HIP Proposal Draft

### PR DESCRIPTION
**Pull request by smcki012**
This is the draft for the Handshake Improvement Proposal process methodology. Derived from the EIP process, which was taken from the BIP process crafted by Amir Taaki, this proposal process takes into account Handshake's expressiveness as an alternative namespace, and makes it more straight-forward when proposing consensus level changes that could affect covenants and the chain's security. Whether developing a process improvement (meta-HIPs), or formalizing an informational standard for Handshake name services and applications, this provides us a way to separate the various changes based on their impact and scope.

### CODE REVIEW

File: HIPs/hip-1.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+For Handshake client implementers, HIPs are a convenient way to track the progress of their implementation. Ideally each implementation maintainer would list the HIPs that they have implemented. This will give end users a convenient way to know the current status of a given implementation or library.
+
+## HIP Types
+
+There are three types of HIP:
+
+- A **Standard Track HIP** describes any change that affects most or all Handshake implementations, such as a change to the network protocol, a change in block or transaction validity rules, proposed naming application standards/conventions, or any change or addition that affects the interoperability of applications using Handshake. Furthermore Standard HIPs can be broken down into the following categories. Standards Track HIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification].
```
Comment (pinheadmz) [inline HIPs/hip-1.md:25]: Again since many HIPs won't necessarily apply to consensus (like name swaps or standards for selling a name) I think we can either add a new section "Application" or rephrase this to be more lenient around "all implementations". We have this whole DNS resolver layer as well that may not be critical for all HNS resolvers.

Comment (pinheadmz) [inline HIPs/hip-1.md:25]: [formal specification] link is missing - did this used to point to ethereum yellow paper or something? i think it can just be removed.

File: HIPs/hip-1.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+  - **Interface** - includes improvements around client [API/RPC] specifications and standards, and also certain language-level standards like method names. The label “interface” aligns with the [interfaces repo] and discussion should primarily occur in that repository before an HIP is submitted to the HIPs repository.
+  - **HRC** - application-level standards and conventions, including contract standards such as naming and domain standards, name registries, URI schemes, library/package formats, and wallet formats.
+- A **Meta HIP** describes a process surrounding Handshake or proposes a change to (or an event in) a process. Process HIPs are like Standards Track HIPs but apply to areas other than the Handshake protocol itself. They may propose an implementation, but not to Handshake's codebase; they often require community consensus; unlike Informational HIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Handshake  development. Any meta-HIP is also considered a Process HIP.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:30]: Let's pick either "meta" or "process" and just go with it instead of having a synonym - this is a specification after all!

File: HIPs/hip-1.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+It is highly recommended that a single HIP contain a single key proposal or new idea. The more focused the HIP, the more successful it tends to be. A change to one client doesn't require an HIP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
+
+An HIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+
+### Special Requirements for Core HIPs
+
+If a **Core** HIP mentions or proposes changes to the Covenent systems, it should refer to the instructions by their mnemonics and define the opcodes of those mnemonics at least once. A preferred way is the following:
+```
+TRANSFER (0xfe)
```
Comment (pinheadmz) [inline HIPs/hip-1.md:41]: TRANSFER is not an opcode, and is not represented by a hex byte. We could use their [type numbers](https://github.com/handshake-org/hsd/blob/master/lib/covenants/rules.js#L43) here but I think that's overkill.

You can mention both OPCODES like bitcoin (not like ethereum) and covenant types here as separate examples maybe. A HIP would be the right place to propose a new covenant type for example.

File: HIPs/hip-1.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+```
+
+## HIP Work Flow
+
+### Shepherding an HIP
+
+Parties involved in the process are you, the champion or *HIP author*, the [*HIP editors*](#hip-editors), and the [*handshake Client Developers*](#client-developers).
```
Comment (pinheadmz) [inline HIPs/hip-1.md:48]: Not sure if we need this line. If we do - "Handshake" should be capitalized and the link doesn't work. Are the client developers different from the HIP editors?

File: HIPs/hip-1.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+
+Before you begin writing a formal HIP, you should vet your idea. Ask the Handshake community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is thus recommended to open a discussion thread on [the Handshake Magicians forum] to do this, but you can also use [one of the Handshake Gitter chat rooms], [the Handshake subreddit] or [the Issues section of this repository].
```
Comment (pinheadmz) [inline HIPs/hip-1.md:50]: Ok my thought on this since we don't have any of these things, and none of these links work -- maybe HIPs that are just vague ideas should start as issues in this repo, and we can organize discussion there. So if a developer has an idea like "using HNS names for login credentials" but no spec yet, they can open an ISSUE (will review your issue_template as well) with a vague proposal. And then the community can help them move forward or abort if for example, there is already prior art or someone else solved the problem already.

File: HIPs/hip-1.md [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+
+Before you begin writing a formal HIP, you should vet your idea. Ask the Handshake community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is thus recommended to open a discussion thread on [the Handshake Magicians forum] to do this, but you can also use [one of the Handshake Gitter chat rooms], [the Handshake subreddit] or [the Issues section of this repository]. 
+
+In addition to making sure your idea is original, it will be your role as the author to make your idea clear to reviewers and interested parties, as well as inviting editors, developers and community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your HIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the work required for implementing a Core HIP will be much greater than for an ERC and the HIP will need sufficient interest from the Handshake client teams. Negative community feedback will be taken into consideration and may prevent your HIP from moving past the Draft stage.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:52]: ERC ?

File: HIPs/hip-1.md [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+
+### Core HIPs
+
+For Core HIPs, given that they require client implementations to be considered **Final** (see "HIPs Process" below), you will need to either provide an implementation for clients or convince clients to implement your HIP. 
+
+The best way to get client implementers to review your HIP is to present it on an AllCoreDevs call. You can request to do so by posting a comment linking your HIP on an [AllCoreDevs agenda GitHub Issue].
```
Comment (pinheadmz) [inline HIPs/hip-1.md:58]: Again, no AllCoreDevs call. Lets just try to keep everything in this github repo for now. Maybe also mention #handshake on IRC? I'm wary of even mentioning telegram channels because they may be ephemeral.

File: HIPs/hip-1.md [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+
+Following is the process that a successful Core HIP will move along:
+
+```
+[ IDEA ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
+```
+
+Each status change is requested by the HIP author and reviewed by the HIP editors. Use a pull request to update the status. Please include a link to where people should continue discussing your HIP. The HIP editors will process these requests as per the conditions below.
+
+* **Idea** -- Once the champion has asked the Handshake community whether an idea has any chance of support, they will write a draft HIP as a [pull request]. Consider including an implementation if this will aid people in studying the HIP.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:84]: [pull request] link missing

File: HIPs/hip-1.md [hunk 9]
```diff
@@ -0,0 +1,271 @@
...
+  * :arrow_right: Draft -- If agreeable, HIP editor will assign the HIP a number (generally the issue or PR number related to the HIP) and merge your pull request. The HIP editor will not unreasonably deny an HIP.
+  * :x: Draft -- Reasons for denying draft status include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the [Handshake philosophy](https://github.com/Handshake/wiki/wiki/White-Paper#philosophy).
```
Comment (pinheadmz) [inline HIPs/hip-1.md:86]: [handshake philosophy] link broken -- do we have a philosophy? maybe https://hsd-dev.org/files/handshake.txt

File: HIPs/hip-1.md [hunk 10]
```diff
@@ -0,0 +1,271 @@
...
+* **Draft** -- Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the HIP to be mature and ready to proceed to the next status. An HIP in draft status must be implemented to be considered for promotion to the next status (ignore this requirement for core HIPs).
+  * :arrow_right: Last Call -- If agreeable, the HIP editor will assign Last Call status and set a review end date (`review-period-end`), normally 14 days later.
+  * :x: Last Call -- A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that HIPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:89]: RSS feed?

File: HIPs/hip-1.md [hunk 11]
```diff
@@ -0,0 +1,271 @@
...
+* **Last Call** -- This HIP will listed prominently on the https://HIPs.Handshake.org/ website (subscribe via RSS at [last-call.xml](/last-call.xml)).
```
Comment (pinheadmz) [inline HIPs/hip-1.md:90]: URL does not exist, RSS does not exist.

File: HIPs/hip-1.md [hunk 12]
```diff
@@ -0,0 +1,271 @@
...
+* **Abandoned** -- This HIP is no longer pursued by the original authors or it may not be a (technically) preferred option anymore.
+  * :arrow_right: Draft -- Authors or new champions wishing to pursue this HIP can ask for changing it to Draft status.
+* **Rejected** -- An HIP that is fundamentally broken or a Core HIP that was rejected by the Core Devs and will not be implemented. An HIP cannot move on from this state.
+* **Superseded** -- An HIP which was previously Final but is no longer considered state-of-the-art. Another HIP will be in Final status and reference the Superseded HIP. An HIP cannot move on from this state.
+
+## What belongs in a successful HIP?
+
+Each HIP should have the following parts:
+
+- Preamble - RFC 822 style headers containing metadata about the HIP, including the HIP number, a short descriptive title (limited to a maximum of 44 characters), and the author details. See [below](https://github.com/Handshake/HIPs/blob/master/HIPS/HIP-1.md#HIP-header-preamble) for details.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:112]: broken link

File: HIPs/hip-1.md [hunk 13]
```diff
@@ -0,0 +1,271 @@
...
+- Abstract - A short (~200 word) description of the technical issue being addressed.
+- Motivation (*optional) - The motivation is critical for HIPs that want to change the Handshake protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the HIP solves. HIP submissions without sufficient motivation may be rejected outright.
+- Specification - The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Handshake platforms (HSD, RSD)
```
Comment (pinheadmz) [inline HIPs/hip-1.md:115]: I dont think we need examples of implementations here

File: HIPs/hip-1.md [hunk 14]
```diff
@@ -0,0 +1,271 @@
...
+- Security Considerations - All HIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life cycle of the proposal. E.g. include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. HIP submissions missing the "Security Considerations" section will be rejected. An HIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.
+- Copyright Waiver - All HIPs must be in the public domain. See the bottom of this HIP for an example copyright waiver.
+
+## HIP Formats and Templates
+
+HIPs should be written in [markdown] format.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:125]: missing link

File: HIPs/hip-1.md [hunk 15]
```diff
@@ -0,0 +1,271 @@
...
+Image files should be included in a subdirectory of the `assets` folder for that HIP as follows: `assets/HIP-N` (where **N** is to be replaced with the HIP number). When linking to an image in the HIP, use relative links such as `../assets/HIP-1/image.png`.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:126]: you can create an empty assets folder for your PR so at least something exists. Maybe just include a small png of the handshake logo (github wont like a totally empty folder)

File: HIPs/hip-1.md [hunk 16]
```diff
@@ -0,0 +1,271 @@
...
+
+## HIP Header Preamble
+
+Each HIP must begin with an [RFC 822](https://www.ietf.org/rfc/rfc822.txt) style header preamble, preceded and followed by three hyphens (`---`). This header is also termed ["front matter" by Jekyll](https://jekyllrb.com/docs/front-matter/). The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:130]: again i kinda prefer the backticks preamble form BIPs, but not gonna die on that hill. I just think its cleaner. and you can reference BIP-1 directly here instead of a 40 year old RFC ;-)

File: HIPs/hip-1.md [hunk 17]
```diff
@@ -0,0 +1,271 @@
...
+
+While an HIP is a draft, a `discussions-to` header will indicate the mailing list or URL where the HIP is being discussed. As mentioned above, examples for places to discuss your HIP include an issue in this repo or in a fork of this repo, [Reddit r/Handshake](https://www.reddit.com/r/Handshake/), and technical asynhronous channels where discussions are frequented. Overtime we will update this document to better inform an HIP author on where to best champion their efforts for acceptance/rough consensus.
+
+No `discussions-to` header is necessary if the HIP is being discussed privately with the author.
+
+As a single exception, `discussions-to` cannot point to GitHub pull requests.
+
+#### `type` header
+
+The `type` header specifies the type of HIP: Standards Track, Meta, or Informational. If the track is Standards please include the subcategory (core, networking, interface, or HRC).
```
Comment (pinheadmz) [inline HIPs/hip-1.md:194]: HRC?

File: HIPs/hip-1.md [hunk 18]
```diff
@@ -0,0 +1,271 @@
...
+
+It occasionally becomes necessary to transfer ownership of HIPs to a new champion. In general, we'd like to retain the original author as a co-author of the transferred HIP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the HIP process, or has fallen off the face of the 'net (i.e. is unreachable or isn't responding to email). A bad reason to transfer ownership is because you don't agree with the direction of the HIP. We try to build consensus around an HIP, but if that's not possible, you can always submit a competing HIP.
+
+If you are interested in assuming ownership of an HIP, send a message asking to take over, addressed to both the original author and the HIP editor. If the original author doesn't respond to email in a timely manner, the HIP editor will make a unilateral decision (it's not like such decisions can't be reversed :)).
+
+## HIP Editors
+
+The current HIP editors are
+
+` * Name (@GithubUser)`
```
Comment (pinheadmz) [inline HIPs/hip-1.md:230]: RIght now this is @pinheadmz and @tynes if you wanna start it off

File: HIPs/hip-1.md [hunk 19]
```diff
@@ -0,0 +1,271 @@
...
+
+The editors don't pass judgment on HIPs. We merely do the administrative & editorial part.
+
+## Style Guide
+
+When referring to an HIP by number, it should be written in the hyphenated form `HIP-X` where `X` is the HIP's assigned number.
+
+## History
+
+This document was derived heavily from [Handshake's HIP-1], which was modified from the BIP process written by Amir Taaki, which in turn was derived from [Python's PEP-0001]. In many places text was simply copied and modified. Although the PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger, they are not responsible for its use in the Handshake Improvement Process, and should not be bothered with technical questions specific to Handshake or the HIP. Please direct all comments to the HIP editors.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:261]: missing links. also this IS HIP-1, probably isnt derived heavily form HIP-1 ;-)

File: HIPs/hip-1.md [hunk 20]
```diff
@@ -0,0 +1,271 @@
...
+
+August 7, 2020: HIP-1 has been improved and will be placed as a PR.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:263]: improved? or created!

File: HIPs/hip-1.md [hunk 21]
```diff
@@ -0,0 +1,271 @@
...
+
+
+### Bibliography
```
Comment (pinheadmz) [inline HIPs/hip-1.md:266]: Can remove or combine with the preceding section that references EIP BIP and PEP

File: ISSUE_TEMPLATE.md [hunk 22]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (pinheadmz) [inline ISSUE_TEMPLATE.md:1]: Github has a notice on this file:

" You are using an old version of issue templates. Please update to the new issue template workflow. Learn more "

https://docs.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

Bob wallet has a good issue template file.

In addition I think issue and pr templates should go in a `.github` directory.

I would recomened TWO issue templates using the new github format:

1) start a proposal discussion - informal proposal with vague details or just a statement of a problem ("how can we sotre blinded bid values on chain in such a way that they can be recovered with bip44?") etc or ("using TXT records to store the name of your favorite movie") so the community can vet the idea at the very beginning, before the author wastes time doing research or writing code

2) an actual issue with an existing HIP or the repo itself ("HIP-1 links are broken") etc.

File: PULL_REQUEST_TEMPLATE.md [hunk 23]
```diff
@@ -0,0 +1,8 @@
+When opening a pull request to submit a new HIP, please use the suggested template: (Link to HIP Template Github)
```
Comment (pinheadmz) [inline PULL_REQUEST_TEMPLATE.md:1]: Link is missing. Also we don't have a bot or a build process, so we can take those details out for now.

File: README.md [hunk 24]
```diff
@@ -1,2 +1,56 @@
-# HIPs
-Handshake Improvement Proposals
+# Handshake Improvement Proposals (HIPs)
+
+
+
+Handshake Improvement Proposals (HIPs) describe standards for the Handshake platform, including core protocol specifications, client APIs, and covenant standards.
```
Comment (pinheadmz) [inline README.md:5]: let's include something like "applications and DNS resolvers" in this list

File: README.md [hunk 25]
```diff
@@ -1,2 +1,56 @@
...
+
+A browsable version of all current and draft HIPs can be found on [Future Site](HSD-Dev.org).
```
Comment (pinheadmz) [inline README.md:7]: site doesnt exist? I say remove.

File: README.md [hunk 26]
```diff
@@ -1,2 +1,56 @@
...
+
+# Contributing
+
+ 1. Review [HIP-1](HIPS/hip-1.md).
+ 2. Fork the repository by clicking "Fork" in the top right.
+ 3. Add your HIP to your fork of the repository. There is a [template HIP here](hip-template.md).
```
Comment (pinheadmz) [inline README.md:13]: these links need tweaking, capitalization is incorrect.

File: README.md [hunk 27]
```diff
@@ -1,2 +1,56 @@
...
+ 4. Submit a Pull Request to Handshake [HIPs repository](https://github.com/handshake-org/HIPs/).
+
+Your first PR should be a first draft of the final HIP. It must meet the formatting criteria enforced by the build (largely, correct metadata in the header). An editor will manually review the first PR for a new HIP and assign it a number before merging it. Make sure you include a `discussions-to` header with the URL to a discussion forum or open GitHub issue where people can discuss the HIP as a whole.
```
Comment (pinheadmz) [inline README.md:16]: no build process, just mention the style guide in HIP1

File: README.md [hunk 28]
```diff
@@ -1,2 +1,56 @@
...
+
+If your HIP requires images, the image files should be included in a subdirectory of the `assets` folder for that HIP as follows: `assets/hip-N` (where **N** is to be replaced with the HIP number). When linking to an image in the HIP, use relative links such as `../assets/hip-1/image.png`.
```
Comment (pinheadmz) [inline README.md:18]: Hm this is redundant with details in HIP1 isnt it? Can we consolidate these files or just keep the details in one place? Maybe the style guide is HIP-1 and the readme can be simplified.

File: README.md [hunk 29]
```diff
@@ -1,2 +1,56 @@
...
+
+Once your first PR is merged, we have a bot that helps out by automatically merging PRs to draft HIPs. For this to work, it has to be able to tell that you own the draft being edited. Make sure that the 'author' line of your HIP contains either your GitHub username or your email address inside <triangular brackets>. If you use your email address, that address must be the one publicly shown on [your GitHub profile](https://github.com/settings/profile).
```
Comment (pinheadmz) [inline README.md:20]: we don't have a bot.

File: README.md [hunk 30]
```diff
@@ -1,2 +1,56 @@
...
+
+When you believe your HIP is mature and ready to progress past the draft phase, you should do one of two things:
+
+ - **For a Standards Track HIP of type Core**, ask to have your issue added to [the agenda of an upcoming Handshake Developer meeting](Link to Meeting Schedule), where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the HIP editors will update the state of your HIP to 'Accepted'.
```
Comment (pinheadmz) [inline README.md:24]: broken link, there is no meeting schedule. how else can we attract attention to a HIP proposal?

File: README.md [hunk 31]
```diff
@@ -1,2 +1,56 @@
...
+
+* **Draft** - an HIP that is undergoing rapid iteration and changes.
+* **Last Call** - an HIP that is done with its initial iteration and ready for review by a wide audience.
+* **Accepted** - a core HIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. The process for Handshake Client Developers to decide whether to encode an HIP into their clients as part of a hard fork is not part of the HIP process. If such a decision is made, the HIP will move to final.
+* **Final (Non-Core)** - an HIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
+* **Final (Core)** - an HIP that the Handshake Client Developers have decided to implement and release in a future hard fork or has already been released in a hard fork. 
+
+# Preferred Citation Format
+
+The canonical URL for HIP that has achieved draft status at any point is at the Handshake-Org HIPs. For example, the canonical URL for HIP-1 is (Link to HIP-001)
```
Comment (pinheadmz) [inline README.md:37]: link is missing. Also my preference is capitalization like "handshake-org" since thats the actual style of the github organization.

File: README.md [hunk 32]
```diff
@@ -1,2 +1,56 @@
...
+
+# Validation
+
+**Not Relant to Handshake HIP Process At This Time**
+
+HIPs must pass some validation tests.  The HIP repository ensures this by running tests using [html-proofer](https://rubygems.org/gems/html-proofer) and [hip_validator](Link to Validator).
```
Comment (pinheadmz) [inline README.md:43]: no validator.

File: README.md [hunk 33]
```diff
@@ -1,2 +1,56 @@
...
+
+It is possible to run the HIP validator locally:
+```sh
+gem install hip_validator
+hip_validator <INPUT_FILES>
+```
+
+# Automerger
```
Comment (pinheadmz) [inline README.md:51]: just remove until it's relevant I think.

File: HIPs/hip-1.md [hunk 34]
```diff
@@ -0,0 +1,271 @@
...
+  - **HRC** - application-level standards and conventions, including contract standards such as naming and domain standards, name registries, URI schemes, library/package formats, and wallet formats.
+- A **Meta HIP** describes a process surrounding Handshake or proposes a change to (or an event in) a process. Process HIPs are like Standards Track HIPs but apply to areas other than the Handshake protocol itself. They may propose an implementation, but not to Handshake's codebase; they often require community consensus; unlike Informational HIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Handshake  development. Any meta-HIP is also considered a Process HIP.
+- An **Informational HIP** describes an Handshake design issue, or provides general guidelines or information to the Handsahke community, but does not propose a new feature. Informational HIPs do not necessarily represent Handshake community consensus or a recommendation, so users and implementers are free to ignore Informational HIPs or follow their advice.
```
Comment (pinheadmz) [inline HIPs/hip-1.md:31]: nit: an/a Handshake

**Comment by smcki012** [comment]
Will review. I was waiting to do a complete revamp this weekend... But, saw others had already begun submitting draft versions and wanted to get started so others like Falci didn't have to wait. If I have sometime I'll amend, but this is a good starting point.

**Comment by pinheadmz** [comment]
Ok, honestly we can just start with a brief README that contains a table of contents and the template submitted by Falci in #2. I don't think we need to spell out every step of the governance process just yet. @tynes @falci and myself all have HIP ideas that we want to publish and the basic template is enough to get started.

**Comment by smcki012** [comment]
You all have it handled. Closing.
